src/api/java/com/minecolonies/api/blocks/AbstractBlockHut.java

```diff
@@ -330,4 +332,10 @@ public boolean needsResearch()
     {
         return needsResearch;
     }
+
+    @Override
+    public void registerBuildingModules(final IBuilding building)
+    {
+
```

Comment:
You might want to consider firing an event here somehow.
This to allow external APIs to inject modules. It does not need to be implemented here yet, just a comment to mark it as a TODO somehow should suffice for this PR.